ledger/src/blockstore.rs

```diff
@@ -1051,6 +1051,26 @@ impl Blockstore {
             });
 
             if erasure_config != erasure_meta.config {
+                let coding_start = erasure_meta.first_coding_index;
+                let coding_end = coding_start + erasure_meta.size as u64;
```

Comment:
hmmm is this `erasure_meta.size` set anywhere? It defaults to 0, and I don't see `ErasureMeta::set_size` called anywhere. 

Maybe a test to see if this logic catches the duplicate and marks the duplicate column?